docs/01-Introduction.md

```diff
@@ -1,6 +1,82 @@
 # 1.0 - Introduction
 
-## 1.1 What is LORIS?
+### 1.1 What is LORIS-MRI?  LORIS-MRI is the backbone of the imaging
+component that makes up LORIS.
+ It is maintained in a separate repository so that it can be installed
+ on the file server and separated from the web server.
+These documents assume you have some knowledge regarding LORIS and
+a functioning installation. For information regarding LORIS itself,
+please consult the [LORIS wiki][1].
 
-## 1.2 How does LORIS work?
+LORIS-MRI is a set of libraries responsible for the insertion,
+visualization and archival of uploaded imaging datasets.
+ It expects an uploaded, compressed file containing
+several [DICOM][2] files, converts this into [MINC][3] and [NII][4]
+files, and then archives these files onto the server. Knowledge of
+these file formats can be helpful, but are not necessary for using or
+installing LORIS-MRI.
 
+LORIS-MRI allows you to easily organize and archive your imaging datasets
+and links them with
+ corresponding behavioral data in LORIS. Scans can be viewed and quality
+ controlled through the user's web browser, facilitating collaboration
+ between radiologists, clinicians and researchers.
+
+### 1.2 How does LORIS-MRI work?  ![user_story](images/user_story.png)
+LORIS-MRI allows multiple ways to upload scans, more details about these
+options can be found [here.](05-PipelineLaunchOptions.md) First, users
+upload a compressed (.tgz, .tar.gz, or .zip) DICOM folder to Imaging
+Uploader.  LORIS requires that the uploaded file name follow the naming
+convention PSCID_CANDID_VISIT-LABEL.  Assuming the upload went through
+successfully, an entry is added to the menu of Imaging Uploader and the
+upload's status throughout the pipeline can then be tracked by the user
+through the Log Viewer.
+
+All DICOM datasets transferred to the DCC must be free of any identifying
```

Comment:
Replace the "transferred to the DCC" by "uploaded via the Imaging Uploader or transferred on the LORIS-MRI server"

Reply:
No everyone uses the term DCC.